WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginWpcomService.java

```diff
@@ -183,6 +185,15 @@ private void setState(LoginPhase loginPhase) {
             mDispatcher.unregister(this);
             stopSelf();
         }
+
+        track();
+    }
+
+    private void track() {
+        Map<String, Object> props = new HashMap<>();
+        props.put("login_phase", mLoginPhase.name());
```

Comment:
can we just make sure `mLoginPhase` is not null here?

Reply:
Makes sense. Added a null check with f83dd70.